src/python/WMCore/Storage/Backends/GFAL2Impl.py

```diff
@@ -102,11 +101,8 @@ def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=No
         args, unknown = parser.parse_known_args(options.split())
 
         if not args.nochecksum:
-            if useChecksum:
-                checksums['adler32'] = "%08x" % int(checksums['adler32'], 16)
-                copyCommandDict['checksum'] = "-K adler32:%s" % checksums['adler32']
-            else:
-                copyCommandDict['checksum'] = "-K adler32"
+            # then calculate the adler32 checksum on both source and destination files
+            copyCommandDict['checksum'] = "--checksum-mode both -K adler32"
```

Comment:
If we are about to always use the `adler32` algorithm for the checksum, please remove this `-K ` argument explanation from the docstring above or at least make a remark that this will always be set to `adler32`